### ISSUE TITLE
Bug: JSON Notifications

### ISSUE THREAD

**Issue by tkhumush** [damus-io/damus#2756]
**What happens**
When I receive notifications, I am seeing the raw JSON object. 

**What I expect to happen**
I expect the notification to show the content of the event properly formatted. 

**Link to noteID, npub**
Not sure. 

**Screenshots/video recording**
![image](https://github.com/user-attachments/assets/abb484b6-369f-4fb0-97b2-e132b518c8df)

** Versions **
Damus version: 1.11(9)
Operating system version: iOS18.1
Device: iPhone 15 Pro

**Comment by alltheseas** [comment]
Looks like a push  notifications regression

**Comment by alltheseas** [comment]
@danieldaquino

**Comment by tkhumush** [comment]
Yeah, it started happening at some point. I wonder if these are notes I sent from notedeck?

**Comment by alltheseas** [comment]
@tkhumush are you still seeing this?

@tyiu can you details here

**Comment by tyiu** [comment]
> @tyiu can you details here

iPhone 14 Pro
iOS 18.2.1
Damus 1.12 (682) 4cf8097

![image](https://github.com/user-attachments/assets/6833cc43-838a-4f59-b37f-5b6023b5153d)

**Comment by alltheseas** [comment]
Lmk if yall still see this
